lib/classes/PluginManager.test.js

```diff
@@ -1586,6 +1586,107 @@ describe('PluginManager', () => {
     });
   });
 
+  describe('#getCommand()', () => {
+    beforeEach(() => {
+      pluginManager.addPlugin(SynchronousPluginMock);
+      pluginManager.serverless.cli.loadedCommands = {
```

Comment:
@pmuens Absolutely! Good call on adding a test for it. 
When adding the `loadedCommands` I would normally put this JS object in a `fixtures` file but it didn't look like this project really uses fixturs. So I put it in line with the test. Let me know if there is a preferred way to do this.

Reply:
Sounds good. Yes, using this inline should be ok :+1: